tbx/services/blocks.py

```diff
@@ -38,8 +40,44 @@ class Meta:
         template = "patterns/molecules/streamfield/blocks/partners_block.html"
 
 
-class ServiceStoryBlock(StoryBlock):
+class CaseStudyStructValue(blocks.StructValue):
+    @cached_property
+    def featured_case_study_image(self):
+        if image := self.get("image"):
```

Comment:
Could you update the editor's guide with details of how the image fallbacks work here (for both image and logo), if you have not done so already? Some help text would also be useful.

Reply:
Thanks, done